sql/moz-fx-data-shared-prod/udf_js/decode_uri_attribution/udf.sql

```diff
@@ -0,0 +1,165 @@
+-- Definition for udf_js.decode_uri_attribution
+-- For more information on writing UDFs see:
+-- https://docs.telemetry.mozilla.org/cookbooks/bigquery/querying.html
```

Comment:
nit: do we usually add this to UDF definitions?

Reply:
It is added by the `/bqetl routine create <dataset>.<name> --udf` command.